.ci/Makefile

```
@@ -113,10 +113,10 @@ yaml-upload: VAULT_AWS_CREDS = secret/cloud-team/cloud-ci/eck-release
 yaml-upload: AWS_ACCESS_KEY_ID = $(shell VAULT_TOKEN=$(VAULT_TOKEN) $(vault) read -address=$(VAULT_ADDR) -field=access-key-id $(VAULT_AWS_CREDS))
 yaml-upload: AWS_SECRET_ACCESS_KEY = $(shell VAULT_TOKEN=$(VAULT_TOKEN) $(vault) read -address=$(VAULT_ADDR) -field=secret-access-key $(VAULT_AWS_CREDS))
 yaml-upload:
-ifndef VERSION
-	$(error VERSION not set to upload YAML to S3)
+ifndef IMG_VERSION
+	$(error IMG_VERSION not set to upload YAML to S3)
 endif
 	@ $(MAKE) \
-		DOCKER_OPTS="-e VERSION=$(VERSION) -e AWS_ACCESS_KEY_ID=$(AWS_ACCESS_KEY_ID) -e AWS_SECRET_ACCESS_KEY=$(AWS_SECRET_ACCESS_KEY)" \
+		DOCKER_OPTS="-e ENABLE_FIPS=$(ENABLE_FIPS) -e VERSION=$(IMG_VERSION) -e AWS_ACCESS_KEY_ID=$(AWS_ACCESS_KEY_ID) -e AWS_SECRET_ACCESS_KEY=$(AWS_SECRET_ACCESS_KEY)" \
```

```suggestion
		DOCKER_OPTS="-e VERSION=$(IMG_VERSION) -e AWS_ACCESS_KEY_ID=$(AWS_ACCESS_KEY_ID) -e AWS_SECRET_ACCESS_KEY=$(AWS_SECRET_ACCESS_KEY)" \
```
Looking at the `ci-internal` target, it does 2 things:

1. calls ci-build-image, which builds `.ci/Dockerfile`, which is contains the needed ci tools, such as golangci-lint, shellcheck, gcloud, eksctl, helm, etc, which doesn't need/use the `ENABLE_FIPS` env var.
2. run `docker run` using the `DOCKER_CMD`, which will be `hack/manifest-upload.sh`, which also doesn't use the `ENABLE_FIPS` env var.

I think this can be removed here.